CMakeLists.txt

```diff
@@ -645,8 +645,8 @@ elseif(UNIX)
 			install(FILES "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/85-sunshine.rules" DESTINATION "${SUNSHINE_ASSETS_DIR}/udev/rules.d")
 			install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service" DESTINATION "${SUNSHINE_ASSETS_DIR}/systemd/user")
 		else()
-			install(FILES "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/85-sunshine.rules" DESTINATION "${CMAKE_INSTALL_LIBDIR}/udev/rules.d")
-			install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service" DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/user")
+			install(FILES "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/85-sunshine.rules" DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d")
+			install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service" DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/systemd/user")
```

Comment:
Here's the 0.15 release:
```
$ dpkg -c ~/Downloads/sunshine.deb  | grep -e udev -e systemd
drwxr-xr-x root/root         0 2022-10-31 11:33 ./usr/lib/x86_64-linux-gnu/systemd/
drwxr-xr-x root/root         0 2022-10-31 11:33 ./usr/lib/x86_64-linux-gnu/systemd/user/
-rw-r--r-- root/root       148 2022-10-31 11:30 ./usr/lib/x86_64-linux-gnu/systemd/user/sunshine.service
drwxr-xr-x root/root         0 2022-10-31 11:33 ./usr/lib/x86_64-linux-gnu/udev/
drwxr-xr-x root/root         0 2022-10-31 11:33 ./usr/lib/x86_64-linux-gnu/udev/rules.d/
-rw-r--r-- root/root        75 2022-10-31 11:24 ./usr/lib/x86_64-linux-gnu/udev/rules.d/85-sunshine.rules
```

CMakeCache.txt from my local build:
```
$ cat CMakeCache.txt | grep CMAKE_INSTALL_LIBDIR
CMAKE_INSTALL_LIBDIR:PATH=lib/x86_64-linux-gnu
//ADVANCED property for variable: CMAKE_INSTALL_LIBDIR
CMAKE_INSTALL_LIBDIR-ADVANCED:INTERNAL=1
```

Reply:
Re: `/usr/share`, that is not a valid installation path. Typically it would be `/usr/local`, which would also be supported by systemd:

https://www.freedesktop.org/software/systemd/man/systemd.unit.html (see Table 1) and https://www.freedesktop.org/software/systemd/man/udev.html.